### PR TITLE
Cloud vision color

### PR DESCRIPTION
**Pull request by deebot10**
## Implemented Features:
-Process started for gathering a hash from the CloudAPI that returns color info about each picture.

## Roadblocks/Logs/Errors:
-

## Testing:
-100%

## Notes/For the future:
-Similar to the categories collected from each picture, how we should go about iterating through the hash and saving the info we want into what tables in the DB.

### CODE REVIEW

File: app/facades/get_art_facade.rb
Comment (kabcouwer) [inline app/facades/get_art_facade.rb:1]: do we need base64 anymore?

File: app/services/cloud_vision_service.rb
Comment (kabcouwer) [inline app/services/cloud_vision_service.rb:4]: can we update this method be updated with:
Faraday.post(url) do |req|
   req.params[:key] = ENV['key']
   req.params['reqests'] =
and so on

File: app/services/cloud_vision_service.rb
Comment (kabcouwer) [inline app/services/cloud_vision_service.rb:9]: same here?

**Review by kabcouwer — APPROVED** [review]
Looks good! Nice work

**Review by marlitas — APPROVED** [review]
Looks good. Also agree with Kim's comments and feedback. Happy to merge in, once those are addressed.

File: spec/facades/get_art_facade_spec.rb
Comment (marlitas) [inline spec/facades/get_art_facade_spec.rb:108]: Okay sweet. Once we merge this in I'll delete the old file that has our API key published.

**Review by marlitas — APPROVED** [review]

**Pull request merged**
